### PR TITLE
Update faqs-slm.mdx

### PR DESCRIPTION
**Pull request by mverde11**
Updating the page to take into account latest releases. Thank you

Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

* What problems does this PR solve?
* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc.
* If your issue relates to an existing GitHub issue, please link to it.

### CODE REVIEW

**Comment by mverde11** [comment]
Updating the page to reflect latest product releases. 

Thank you

File: src/content/docs/service-level-management/faqs-slm.mdx [hunk 1]
```diff
@@ -54,6 +99,14 @@ To view the queries that define an SLI, you can click on the **...** menu on the
   src={slmEditMenu2}
 />
 
+You can also finf them through the summary page, by clicking Edit.
```
Comment (akristen) [inline src/content/docs/service-level-management/faqs-slm.mdx]:
```suggestion
You can also find them through the summary page, by clicking Edit.
```

**Review by akristen — APPROVED** [review]

**Comment by akristen** [comment]
Thanks for updating this content!

**Comment by mverde11** [comment]
Thank you for your help =)

**Pull request merged**